src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -5195,10 +5224,23 @@ void *wm_vuldet_main(wm_vuldet_t * vuldet) {
         }
 
         if ((curr_time - vuldet->last_scan >= vuldet->scan_interval)) {
+            if (!VU_TASKS_QUEUE_FLAGS[VU_INTERVAL_SCAN]) {
+                vu_task_ctx_t *task = create_task("%lu %d %d", 1, VU_INTERVAL_SCAN, WM_TASK_PENDING);
```

Comment:
Even when the interaction with the task manager isn't ready yet, we could represent the workflow of trying to generate a **task_ID** or pushing the task without one.

Reply:
Added a dummy function that represents the communication with the task manager that set an ID between 1-10.